core/build.gradle

```diff
@@ -49,13 +49,22 @@ dependencies {
     implementation group: 'com.opencsv', name: 'opencsv', version: '5.7.1'
     implementation group: 'org.roaringbitmap', name: 'RoaringBitmap', version: '0.7.17'
     implementation group: 'com.google.guava', name: 'guava', version: '32.0.1-jre'
-    implementation group: 'org.apache.arrow', name: 'arrow-vector', version: '10.0.1', {
+
+    def arrowExclusions = {
         exclude group: 'com.fasterxml.jackson.core', module: 'jackson-core'
         exclude group: 'com.fasterxml.jackson.core', module: 'jackson-annotations'
         exclude group: 'com.fasterxml.jackson.core', module: 'jackson-databind'
         exclude group: 'io.netty', module: 'netty-common'
     }
 
+    implementation group: 'org.apache.arrow', name: 'arrow-vector', version: '12.0.1', arrowExclusions
+    implementation group: 'org.apache.arrow', name: 'arrow-memory-netty', version: '12.0.1', arrowExclusions
```

Comment:
Don't forget to update LICENSES :)

Reply:
This puzzled me cause I thought the CI was checking for those because of the work you did. I think you were right we need to add an extra step: https://github.com/neo4j/apoc/pull/451

Reply:
Hmm this is weird, because I tested it out by pushing with missing parts and it had worked then 🤔